hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -306,16 +317,106 @@ public boolean isInlineCompactionEnabled() {
       return !continuousMode && !forceDisableCompaction
           && HoodieTableType.MERGE_ON_READ.equals(HoodieTableType.valueOf(tableType));
     }
+
```

Comment:
Question : Any specific reason why equals/hashcode are added ? Was it added as part of general  best-practices ?

Reply:
Yes I wanted to add equals() for better testability and hence added hashcode() as both go hand-in-hand